beeswax/logs/ad_log.proto

```diff
@@ -14,6 +14,7 @@ syntax = "proto2";
 package logs;
 
 option java_package = "com.beeswax.logs";
+option cc_enable_arenas = true;
```

Comment:
What does this line do?

Reply:
Could ask Dan Thompson, he seems to be the originator